### PR TITLE
HT-9 All style

### PR DESCRIPTION
**Pull request by laurgao**
This pull request consists mostly of CSS changes.

- Logo for navbar + favicon
- Main changed pages:
image (1)
image (2)
image

### CODE REVIEW

File: components/CreateEventModal.tsx [hunk 1]
```diff
@@ -98,6 +98,9 @@ const CreateEventModal = ({isOpen, setIsOpen, schoolId, iter, setIter}: {
                 setValue={setDescription}
                 placeholder="Unfortunately, this is not a bowl of HOSA swag."
             />
+            <p className="text-gray-400 text-sm -mt-8">
+                You can use markdown like **bold**, *italic*, [link text](https://your-url.com/), and ~~strikethrough~~. <a href="https://www.markdownguide.org/basic-syntax/" className="underline">Full markdown guide</a>
```
Comment (ameletto) [inline components/CreateEventModal.tsx:102]: very epic

Comment (ameletto) [inline components/CreateEventModal.tsx:102]: very epic

File: components/SEO.tsx [hunk 2]
```diff
@@ -15,11 +15,11 @@ export default function SEO({
     let openGraph = {
         title: fullTitle,
         description: description,
-        url: "https://your-domain.com" + router.asPath,
+        url: "https://hosatinder.vercel.app" + router.asPath,
```
Comment (ameletto) [inline components/SEO.tsx:18]: what does this do?

**Review by ameletto — APPROVED** [review]
feedback: 

1) i am very much in love idk what I would do without you <3 laura is speed I'm literally willing to give you a position at HOSA -- head of web dev just for this

few questions I have that if I was more seasoned i may be able to figure out from the code but unfortunately, i am not:
- do labels automatically show up on cards or do we have to copy and paste the code every time?
- for the tell us about yourself page, do the inputs go to the user schema? and do they do anything? 

more feedback:
- it would be nice if you could show the choose top 3 page without the spots filled in
- also keep font consistent :)

File: components/CreateEventModal.tsx (hunk 1 above)
Comment (laurgao) [reply components/CreateEventModal.tsx:102]: <3

File: components/SEO.tsx (hunk 2 above)
Comment (laurgao) [reply components/SEO.tsx:18]: I think it's just so the search engine optimization stuff knows that the URL of the website is? it was part of samson's template

**Comment by laurgao** [comment]
HI AMY thank you for the feedback

To answer your questions:
- labels show up with every instance of the EventCard component
- the inputs do get stored on mongodb (I think, will have to verify that) and right now they don't do anything. There's a [story](https://hosatinder.atlassian.net/jira/software/projects/HT/boards/1?selectedIssue=HT-3) to sort the tinder cards based on ppl's tags + previous events

Re feedback:
- for font i think i'll make everything raleway by default
- the top 3 spots not filled in will take a while bc i'll have to revamp the data structure that events are stored in, how urgent/important do you think it is?

**Comment by laurgao** [comment]
im going to make a task on Jira for the drag and drop stuff, going to merge this first

**Pull request merged**